### PR TITLE
docs(nuxt): Manual setup for beta

### PR DESCRIPTION
**Pull request by s1gr1d**
## DESCRIBE YOUR PR

Updates the docs for the `@sentry/nuxt` beta release

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [x] Urgent deadline (GA date, etc.): 18. Oct
- [ ] Other deadline: 
- [ ] None: Not urgent, can wait up to 1 week+

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

### CODE REVIEW

File: platform-includes/getting-started-primer/javascript.nuxt.mdx [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <Alert level="warning">
-  This SDK is considered **experimental and in an alpha state**. It may experience breaking changes. Please reach out on
+  This SDK is **in an beta state**. It may experience breaking changes. Please reach out on
```
Comment (Lms24) [inline platform-includes/getting-started-primer/javascript.nuxt.mdx:2]: beta technically means, we don't break APIs any more. 

```suggestion
  This SDK is **in an beta state**. It may still experience changes in behavior. Please reach out on
```

File: platform-includes/getting-started-sourcemaps/javascript.nuxt.mdx [hunk 2]
```diff
@@ -24,3 +24,12 @@ export default defineNuxtConfig({
   }
 });
 ```
+
+Nuxt sets default values for the source maps ([Nuxt docs](https://nuxt.com/docs/api/nuxt-config#sourcemap)) and the SDK currently keeps the source maps settings if they are explicitly defined.
+To enable this source maps setting, add the following code to you Nuxt config:
```
Comment (Lms24) [inline platform-includes/getting-started-sourcemaps/javascript.nuxt.mdx:29]: I think as a first-time reader, I'd a bit confused here. We start out with an explanation but do I have to configure something explicitly to get source maps working?  Maybe rephrase it a bit by telling users what they should set first. Any explanations that are necessary, we can make, probably underneath the code block. wdyt?

File: platform-includes/getting-started-config/javascript.nuxt.mdx [hunk 3]
```diff
@@ -44,40 +44,33 @@ Sentry.init({
...
 import dotenv from 'dotenv';
-import * as Sentry from '@sentry/nuxt';
 
 dotenv.config();
 
-Sentry.init({
-  dsn: "___PUBLIC_DSN___"
-});
+// ... rest of the file
 ```
```
Comment (lforst) [inline platform-includes/getting-started-config/javascript.nuxt.mdx:70]: No need to act now - just sharing my thoughts: I don't think how users can pass env vars into their app shouldn't be in our documentation but idk.

**Review by lforst — APPROVED** [review]

File: platform-includes/getting-started-config/javascript.nuxt.mdx (hunk 3 above)
Comment (s1gr1d) [reply platform-includes/getting-started-config/javascript.nuxt.mdx:70]: I added it because there's no need to do those extra steps when using Nuxt. Usually, you can use `useRuntimeConfig()`, where your env vars are set and `process.env` can be used without explicitly installing `dotenv`. 

However, as the server-side config is loaded before any Nuxt code is loaded, the env setup has to be done manually.
